src/main/java/org/prgrms/kdtspringdemo/view/console/Console.java

```diff
@@ -0,0 +1,112 @@
+package org.prgrms.kdtspringdemo.view.console;
+
+import org.prgrms.kdtspringdemo.customer.constant.CustomerCommand;
+import org.prgrms.kdtspringdemo.view.constant.MainCommandType;
+import org.prgrms.kdtspringdemo.voucher.constant.VoucherCommand;
+import org.prgrms.kdtspringdemo.voucher.constant.VoucherType;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.Scanner;
+import java.util.UUID;
+
+public class Console {
+    private static final Logger logger = LoggerFactory.getLogger(Console.class);
+    private static final String IS_NUMERIC_EXCEPTION_MESSAGE = "바우처의 할인양은 숫자를 입력해주세요.\n";
+    private static final String UUID_FORMAT_EXCEPTION_MESSAGE = "UUID 형식이 잘못 입력되었습니다.";
+
+    private final Scanner scanner = new Scanner(System.in);
+
+    public void printMessage(String message) {
+        System.out.print(message);
+    }
+
+    public MainCommandType inputMainCommand(Message message) {
+        printMessage(message.getText());
+
+        return MainCommandType.findCommandType(scanner.nextLine());
+    }
+
+    public VoucherCommand inputVoucherCommand(Message message) {
+        printMessage(message.getText());
+
+        return VoucherCommand.findVoucherCommand(scanner.nextLine());
+    }
+
+    public VoucherType chooseVoucherType(Message message) {
+        printMessage(message.getText());
+
+        return VoucherType.findVoucherType(scanner.nextLine());
+    }
+
+    public Long inputAmountByVoucher(Message message) {
+        printMessage(message.getText());
+        String inputAmount = scanner.nextLine();
+        validateAmountIsNumeric(inputAmount);
+
+        return Long.parseLong(inputAmount);
+    }
+
+    private void validateAmountIsNumeric(String amount) {
+        try {
+            Long.parseLong(amount);
+        } catch (NumberFormatException e) {
+            logger.error("원인 : {} -> 에러 메시지 : {}", amount, IS_NUMERIC_EXCEPTION_MESSAGE);
+            throw new IllegalArgumentException(IS_NUMERIC_EXCEPTION_MESSAGE);
+        }
+    }
+
+    public UUID inputVoucherId(Message message) {
+        printMessage(message.getText());
+        String userVoucherId = scanner.nextLine();
+        validateVoucherId(userVoucherId);
```

Comment:
목적에 따라서 개행처리를 해주시면 좋을 것 같아요 ! (개인취향~)
```suggestion
        printMessage(message.getText());
        
        String userVoucherId = scanner.nextLine();
        validateVoucherId(userVoucherId);
```

Reply:
- [343dbe8](https://github.com/prgrms-be-devcourse/springboot-basic/pull/838/commits/343dbe849ebf92cf5501c159082bc1a95bf11eca) 피드백 반영

콘솔에서도 input과 output이라는 목적으로 나눌 수 있네요!!
목적을 세분화하도록 노력하겠습니다!! 감사합니다~~!!